libnavigation-core/src/main/java/com/mapbox/navigation/core/telemetry/MapboxNavigationTelemetry.kt

```diff
@@ -180,16 +185,19 @@ internal object MapboxNavigationTelemetry {
         when (sessionState) {
             IDLE, FREE_DRIVE -> {
                 sessionStop()
+                eventsSessionId = obtainUniversalUniqueIdentifier()
```

Comment:
This can be moved up from `when` as it's the same for all cases. Unless we'd like to clear it for `IDLE`?

Reply:
What do you mean? BTW that's outdated. `sessionId` is returned as part of the `NavigationSessionState` from `NavigationSessionStateObserver`.

Reply:
get it 👍